reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -466,6 +487,45 @@ public String toString() {
 					", pendingAcquireTimeout=" + pendingAcquireTimeout +
 					'}';
 		}
+
+		static final class DelegatingAllocationStrategy implements reactor.pool.AllocationStrategy {
```

Comment:
not sure why a `DelegatingAllocationStrategy` is necessary, if the strategies can be deep-copied. can you clarify?

Reply:
The current `reactor.netty.resources.ConnectionProvider.AllocationStrategy` is not `reactor.pool.AllocationStrategy` so we need to provide `reactor.pool.AllocationStrategy`.

Reply:
ah, sorry for missing that 🤦